node/blockchain/inner_models/signed_change_request_message/coin_transfer.py

```diff
@@ -21,5 +21,8 @@ class CoinTransferSignedChangeRequestMessage(SignedChangeRequestMessage):
     txs: list[CoinTransferTransaction]
     type: Type = Field(default=Type.COIN_TRANSFER, const=True)  # noqa: A003
 
-    def get_total_amount(self):
+    def get_total_amount(self) -> int:
         return sum(tx.amount for tx in self.txs)
+
+    def get_total_amount_by_recipient(self, recipient: AccountNumber, is_fee: bool = False) -> int:
+        return sum([tx.amount for tx in self.txs if tx.is_fee is is_fee and tx.recipient == recipient])
```

Comment:
`sum(tx.amount for tx in self.txs if tx.is_fee is is_fee and tx.recipient == recipient)`

@olegtropinin please, remember it, so I do not have to make similar comments again

Reply:
Thank you for noticing. I will.